Felienne/hedy
# [BUG FIX] Fixes color off error message on student invite error

**Description**
In this PR we fix an issue where the color of the error message when inviting a student to a class was incorrect.

**Fix for**
This PR fixes #1851

**How to test**
Make sure you are a teacher, try to invite a non-existing user to a class and notice that the modal alert is red instead of the current green.
